draft-demarco-status-attestations.md

```diff
@@ -435,10 +438,10 @@ The Wallet Instance that provides the Status Attestations MUST include in the
 related Digital Credential.
 
 The Verifier that receive a Digital Credential supporting the Status Attestation,
-SHOULD:
+MUST:
 
 - Decode and validate the Digital Credential;
-- check the presence of `status.status_attestation` in the Digital Credential. If true, the Verifier SHOULD:
+- check the presence of `status.status_attestation` in the Digital Credential. If true, the Verifier MUST:
```

Comment:
it is a SHOULD since a RP may check the revocation or not, depending by different factors.

there are some credentials that even if expired doesn't mean that are not usable, eg: age over.

then for RP's viewpoint it is a SHOULD and not a MUST

Reply:
Okay, this sounds reasonable to me. However, somehow we should include this inside the draft, otherwise having should would be strange if you agree.